recipe/build.sh

```diff
@@ -12,8 +15,16 @@ else
       export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0"
   elif [[ ${cuda_compiler_version} == 11.1* ]]; then
       export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
```

Comment:
Can drop anything before CUDA 11.2+ as conda-forge only builds for CUDA 11.2+ ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ). Should help keep this more manageable